README.md

```diff
@@ -1,2 +1,4 @@
 # bubble-chat
 visitor interaction project during my architecture exhibition
+
+notion page : https://www.notion.so/Exhibition-Interaction-Project-23-02-23-06-28d66e5152134bda83007e629b749905?pvs=4
```

Comment:
It would look better as a markdown hyperlink, i.e

```
[Notion Page](notion_url)
```

Reply:
thank you 

> you're kind
`[Notion page]`